modules/dfpAdServerVideo.js

```diff
@@ -55,9 +56,23 @@ const defaultParamConstants = {
  *   demand in DFP.
  */
 export default function buildDfpVideoUrl(options) {
+  if (!options.params && !options.url) {
+    logError(`A params object or a url is required to use pbjs.adServers.dfp.buildVideoUrl`);
+    return;
+  }
+
+  if (options.params && options.url) {
+    logError(`Passing both a params object and a url to pbjs.adServers.dfp.buildVideoUrl is invalid.`);
```

Comment:
Is there any reason we aren't supporting this? It seems legitimate to parse the URL, overwrite any params, stringify the URL, and then return it.